### ISSUE TITLE
Add publisher/journal: American Chemical Society

### ISSUE THREAD

**Issue by mariaguerreiro** [datadryad/dryad-product-roadmap#3273]
**Name to appear on membership page:** 
American Chemical Society

**Payment plan:**
Annual membership (started in Jan 2024) + quarter DPCs

**Journals included:** 
Journal of Organic Chemistry
Journal of Organic Letters
ACS Organic and Inorganic Au

**Email contacts to receive submission notices: (note whether they receive all notices, or different lists for different journals)**
@ryscher I guess it's worth me asking ACS directly how they'd like to handle that - wdyt?

**Email contacts to receive publication notices: (note whether they receive all notices, or different lists for different journals)** 
@ryscher I guess it's worth me asking ACS directly how they'd like to handle that - wdyt?

**ORCIDs of users who need access to admin dashboard: (they must log in to Dryad to set up their account before we can give them access, note whether each user administers a subset of journals)**
Kevin Thomas, Product Manager, Data Products
Mitchell Bakos, Director of Publications Development
NB: I will ask if they want anyone else to have access to it

**Email contacts to discuss integration of journal metadata with Dryad:**
Kevin Thomas, Product Manager, Data Products, ACS
Mitchell Bakos, Director of Publications Development, ACS

Development tasks:
- [x] add member name to membership page
- [x] update payment settings
- [ ] update journal notification settings
- [x] add journal administrators
- [x] start integration process (track on new ticket)

**Comment by mariaguerreiro** [comment]
@ryscher Can we add ACS to the list of members on the website while we finalise the integration, please? Thanks!

**Comment by ryscher** [comment]
Journals are added and payment settings updated.
